swarm/tools/runs_gc.py

```
@@ -131,26 +132,27 @@ def discover_all_runs() -> List[RunInfo]:
 
     # Examples (always preserved)
     if EXAMPLES_DIR.exists():
-        for entry in EXAMPLES_DIR.iterdir():
+        for entry in os.scandir(EXAMPLES_DIR):
             if entry.is_dir() and not entry.name.startswith("."):
                 if entry.name not in seen:
```

Ruff (0.15.15)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)